src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/readytolist/ReadyToListAboutToSubmitHandler.java

```diff
@@ -25,24 +37,48 @@ public boolean canHandle(CallbackType callbackType, Callback<SscsCaseData> callb
     }
 
     @Override
-    public PreSubmitCallbackResponse<SscsCaseData> handle(CallbackType callbackType, Callback<SscsCaseData> callback,
-                                                          String userAuthorisation) {
+    public PreSubmitCallbackResponse<SscsCaseData> handle(CallbackType callbackType, Callback<SscsCaseData> callback, String userAuthorisation) {
         if (!canHandle(callbackType, callback)) {
             throw new IllegalStateException("Cannot handle callback.");
         }
-        final CaseDetails<SscsCaseData> caseDetails = callback.getCaseDetails();
-        final SscsCaseData sscsCaseData = caseDetails.getCaseData();
 
-        PreSubmitCallbackResponse<SscsCaseData> callbackResponse = new PreSubmitCallbackResponse<>(sscsCaseData);
+        final CaseDetails<SscsCaseData> caseDetails = callback.getCaseDetails();
+        final SscsCaseData sscsCaseData = caseDetails.getCaseData(); //Maybe handle in each method instead of HEAD
+        if(schedulingAndListingFeature){
+            boolean listAssist = checkIfListAssist(sscsCaseData);
+            if(listAssist){
+                sscsCaseData.setHearingRoute("ListAssist");
+                sscsCaseData.setHearingState(StateOfHearing.HEARING_CREATED);
+                PreSubmitCallbackResponse<SscsCaseData> callbackResponse = new PreSubmitCallbackResponse<>(sscsCaseData); //Placeholder
+                return callbackResponse;
+            }else{
+                return handleCallbackResponse(sscsCaseData);
+            }
+        }else{
+            return handleCallbackResponse(sscsCaseData);
+        }
+    }
 
-        log.info(String.format("createdInGapsFrom is %s for caseId %s",
-                sscsCaseData.getCreatedInGapsFrom(), sscsCaseData.getCcdCaseId()));
+    private boolean checkIfListAssist(SscsCaseData sscsCaseData){
+        String region = "SSCS " + sscsCaseData.getRegion();
+        String getIsListAssist = regionalProcessingCenterService.getRegionalProcessingCenterMap().entrySet().stream().
```

Comment:
You should be able to do regionalProcessingCenterService.get(region), check if the value exists then getListAssist on it. With the other changes you will have a boolean to return. If the the value doesn't exist for whatever reason, default to false.

Reply:
EDIT: Should default to true